content/participants/ttjc@shoaibsayyed/index.md

```diff
@@ -31,15 +31,46 @@ It would be good if someone guides me with Firebase(like Authentication) and Rea
 ### how to contact me?
 
 - [Twitter](https://twitter.com/shoaib0311)
-- [LinkedIn](https://www.linkedin.com/in/shoaib-sayyed-522a92163/)
+- [LinkedIn](https://www.linkedin.com/in/shoaib-sayyed/)
 
 ### projects
 
-I will update this section soon with real links. Keep Checking...
+#### 1. Quote - Share Your Thoughts
+
+- [Live URL](https://share-quotes.herokuapp.com/)
+- [Repository on GitHub](https://github.com/shoaib-03/Quote-App)
+
+Hello, Quote App is a **Multi-user** Web App where user can share Quotes with each other. I have created this project using **MERN Stack**. In this app, User can **Create**, **Like**, **Comment** and **Share** their Posts. It has features like **Follow** and **Unfollow** other users, **Deleting Your Posts** and **Comments** and also user can **Update** their Profile.
+
+- **Frontend:** React, React-Router, React-Router-DOM, React-Hooks
+- **Backend:** Node, Express
+- **Database:** MongoDB Atlas
+- **Deployed On:** Heroku
+- **Packages:** Mongoose, jsonwebtoken, bcrypt.js, moment, etc..
+
+I will add more project in this section soon. Keep Checking Please...
 
 ### blogs
 
-Coming Soon...
+#### - 8 Must Know Javascript Array Methods
+
+Hello, I wrote my first blog on "8 Must Know Javascript Array Methods". The reason behind I choose this topic is whenever I do programming Javascript, I always encountered arrays and use any array method. So I thought why not make a of Array Methods that I used generally while programming. That is why came up this blog.
```

Comment:
Hey! @shoaib-03 a small mistake here:
* So i thought why not make a __ of Array Methods ...